handbook/people-ops/benefits-and-perks.md

```diff
@@ -12,15 +12,19 @@ We have a flexible vacation policy with mandatory annual vacation time that enco
 
 ## 🏥 Medical
 
-We offer high-quality medical and dental care for all full-time employees and their dependents.
+We offer high-quality medical, dental, and vision care for all full-time employees and their dependents. We cover 99% to 100% of premiums for employees and all of their dependents for each.
```

Comment:
```suggestion
We offer high-quality medical, dental, and vision care for all US-based full-time employees and their dependents. We cover 99% to 100% of premiums for employees and all of their dependents for each.
```

Quick suggestion based on following the convention in "Family friendly," to make it unambiguous for remote teammates.